src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift

```diff
@@ -1317,20 +447,22 @@ class HealthCertificateValidationServiceValidationTests: XCTestCase {
 			XCTFail("report must not be nil")
 			return
 		}
-		XCTAssertEqual(error, .NO_NETWORK)
```

Comment:
Is the .NO_NETWORK test moved somewhere else?

Reply:
The .`NO_NETWORK` does not existing anymore for the validation service, the error is now encapsulated into the `DCCDownloadRulesError` and is tested in the tests for the resource.
See here: 
https://github.com/corona-warn-app/cwa-app-ios/blob/d3feafff70f31847196b61763b3fa8a22ea0d40e/src/xcode/ENA/ENA/Source/HTTPClient/Resources/__tests__/DCCRulesResourceTests.swift#L307